cpp/cmake/mrd-config.cmake

```diff
@@ -0,0 +1,21 @@
+include(CMakeFindDependencyMacro)
+
+set(ISMRMRD_MINIMUM_VERSION "1.13.4")
```

Comment:
I suppose this is temporary since we are making conversion tools? But it does feel like the old version is now an explicit dependency?